assembly/overworld_scripts/rhodanzi_city/rhodanzi_trainer_school.s

```diff
@@ -139,3 +133,51 @@ EventScript_RhodanziTrainerSchool_BasicCourse_TypeStudent_QuizRejected:
 EventScript_TypeStudentQuizComplete:
     msgbox gText_RhodanziTrainerSchool_BasicCourse_TypeStudent_QuizComplete MSG_FACE
     return
+
+# TODO: Basic course dialog
```

Comment:
nitpicky: you should be consistent with how you are declaring comments. Here you are using # and in other places you are doing // and here you are doing #

Reply:
Agreed to do in next push

Reply:
Agreed to do in next push